crates/tx5/Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords = ["holochain", "holo", "p2p", "webrtc", "networking"]
 categories = ["network-programming"]
 
 [features]
-default = [ "backend-go-pion" ]
+default = [ "backend-libdatachannel" ]
```

Comment:
WARNING: I'm sneaking in a change to default to libdatachannel here, since we'll have to bump the version to 0.3 anyways! : )